vita3k/gui/src/settings_dialog.cpp

```diff
@@ -407,7 +407,7 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
             }
             ImGui::SameLine();
         } else {
-            ImGui::TextColored(GUI_COLOR_TEXT, "No modules present.\nPlease download and install the last firmware.");
+            ImGui::TextColored(GUI_COLOR_TEXT, "No modules present.\nPlease download and install the PS Vita firmware.");
```

Comment:
i said here, keep LAST firmware

Reply:
Any firmware is usable in the emulator, also how can someone not install the latest when the button only allows you to download the latest one